src/execution_plan/ops/op_delete.c

```diff
@@ -105,10 +105,13 @@ static void _DeleteEntities
 		// NOTE: delete edges before nodes
 		// required as a deleted node must be detached
 
+		MATRIX_POLICY policy = Graph_GetMatrixPolicy(op->gc->g);
+		Graph_SetMatrixPolicy(op->gc->g, SYNC_POLICY_NOP);
```

Comment:
Pardon me, please, for this question, but where can I read up on these flags? The enum it is in doesn't contain any documentation. I was wondering when I should use one flag or another and what they change when used.

Is this also a part of improving the performance?

Reply:
you can look at the documentation at Graph_SetMatrixPolicy
this is the performance improvement don't sync the matrix during edge deletion

Reply:
Thanks !